client/packages/invoices/src/InboundShipment/DetailView/modals/utils.ts

```diff
@@ -25,10 +23,10 @@ const createDraftInboundLine = ({
     totalBeforeTax: 0,
     id: FnUtils.generateUUID(),
     invoiceId,
-    sellPricePerPack: 0,
+    sellPricePerPack: item?.itemStoreJoin?.defaultSellPricePerPack ?? 0,
```

Comment:
Is the intended behaviour that this applies to transfer lines as well? Currently if I create a transfer, even if I don't set a sell price as a supplier, it arrives in my store via inbound shipment as 0.00 cost and sell price?

Reply:
Also, I think not from your changes but related:

Create outbound with pack size > 1, set sell price:
<img width="911" height="374" alt="Screenshot 2025-07-28 at 2 21 27 PM" src="https://github.com/user-attachments/assets/360ac3bc-e939-4633-b305-65bdb62516f2" />

Inbound store has pack to 1 - it updates the cost price, but copies the sell price per pack without accounting for pack size?
<img width="761" height="262" alt="Screenshot 2025-07-28 at 2 22 02 PM" src="https://github.com/user-attachments/assets/ba4ea749-886a-4269-a72e-b88abd174a92" />

Reply:
What do you mean? 0.00 cost and sell price? If you don't have a sell price set then why would it be anything but 0.0?

Reply:
If there is a cost price, sell price defaults to be that value (I guess so you don't make a loss on it?)

Reply:
Oh wait sorry, you mean for the one above - I was asking, if I have a default sell price for an item, should incoming lines by transfer also have that default sell price set? Currently its $0.00